.github/ISSUE_TEMPLATE/5_performance_speed.md

```diff
@@ -60,10 +60,31 @@ assignees: ''
 **Target OS version/browser:**
 **Devices:**
 
-## Logs
+
+
+<details>
+<summary>Code sample</summary>
+
+<!--
+     Please create a minimal reproducible sample that shows the problem
+     and attach it below between the lines with the backticks.
+
+     Without this we will unlikely progress on the issue, and thus will have
+     to close it.
```

Comment:
this should probably be phrased in a way that says we're unlikely to be _able_ to progress on the issue, and that we will _regretfully_ have to close it, or some such.